index.html

```diff
@@ -45,6 +45,25 @@
     </div>
 </div>
 
+
+<div class="csh-hacks-wrapper">
+    <div class="container spaced" id="homepage-about">
+        <div class="row">
+            <div class="col-12 col-lg-6">
+                        <h2>September 26 - 27, 2020</h2>
+                        <p>CSH Hacks is an event where developers, designers, engineers and more come together to work on their ideas, solve challenges, learn new skills, and network with companies & each other. CSH Hacks is open to all majors and universities!</p>
+                        <a class="btn btn-primary" target = "_blank" href="https://s3.csh.rit.edu/cshacks-assets/csh_hacks_logo.svg">Learn More</a>
```

Comment:
Why does your link to learn more point to the logo? It should point to `hacks.csh.rit.edu`.

Reply:
`&` -> `&amp;`